pyscitt/pyscitt/client.py

```diff
@@ -363,20 +363,24 @@ def get_did_document(self, did: str) -> dict:
 
     @overload
     def submit_claim(
-        self, claim: bytes, *, skip_confirmation: Literal[False] = False
+        self, claim: bytes, *, skip_confirmation: Literal[False] = False, headers=None
     ) -> Submission:
         ...
 
     @overload
     def submit_claim(
-        self, claim: bytes, *, skip_confirmation: Literal[True]
+        self, claim: bytes, *, skip_confirmation: Literal[True], headers=None
     ) -> PendingSubmission:
         ...
 
     def submit_claim(
-        self, claim: bytes, *, skip_confirmation=False
+        self, claim: bytes, *, skip_confirmation=False, headers=None
     ) -> Union[Submission, PendingSubmission]:
-        headers = {"Content-Type": "application/cose"}
+        request_header = {"Content-Type": "application/cose"}
+
+        if headers:
+            request_header.update(headers)
```

Comment:
You don't use `request_header` below. It might be cleaner to stick with a single variable. Maybe something like:
```py
if headers is None:
  headers = {}
headers["content-type"] = "..."
```

Reply:
@letmaik this has the side effect of modifying the caller's dictionary, which could end up being suprising.

Reply:
@ryazhang-microsoft You marked this as resolved but the `request_header` variable you introduced is still not used anywhere. Please also confirm where this new keyword argument is used.